lib/rucio/common/utils.py

```diff
@@ -260,11 +260,6 @@ def adler32(file):
     adler = 1
 
     can_mmap = False
-    # try:
-    #    with open(file, 'r+b') as f:
-    #        can_mmap = True
-    # except:
-    #    pass
```

Comment:
It would be best to have this removal in a separate commit, as it doesn't match the scope of its current commit ("Adding logging to otherwise bare try/except/pass's")